plugins/hypervisors/vmware/test/com/cloud/hypervisor/vmware/resource/VmwareResourceTest.java

```diff
@@ -216,6 +216,20 @@ public void testScaleVMF1() throws Exception {
     }
 
     @Test
+    public void testReplaceNicsMacSequenceInBootArgs() throws Exception {
```

Comment:
Do you need this `throws Exception` here?
It does not seem o be required by any of the method calls you have here.